docs/CORE_DEV_GUIDE.md

```diff
@@ -0,0 +1,197 @@
+it gives the contributor a choice in whether to accept the changes.
+
+Unless you know that a contributor is experienced with git, don’t
+ask for a rebase when merge conflicts arise. Instead, rebase the
+branch yourself, force-push to their branch, and advise the contributor to force-pull.  If the contributor is
+no longer active, you may take over their branch by submitting a new pull
+request and closing the original, including a reference to the original pull
+request. In doing so, ensure you communicate that you are not throwing the
+contributor's work away!
+
+### Merge Only Changes You Understand
+
+*Long-term maintainability* is an important concern.  Code doesn't
+merely have to *work*, but should be *understood* by multiple core
+developers.  Changes will have to be made in the future, and the
+original contributor may have moved on.
+
+Therefore, *do not merge a code change unless you understand it*. Ask
+for help freely: we can consult community members, or even external developers,
+for added insight where needed, and see this as a great learning opportunity.
+
+While we collectively "own" any patches (and bugs!) that become part
+of the code base, you are vouching for changes you merge.  Please take
+that responsibility seriously.
+
+## Further resources
+
+As a core member, you should be familiar with community and developer
+resources such as:
+
+-  Our [contributor guide](CONTRIBUTING.md).
+-  Our [code of conduct](CODE_OF_CONDUCT.md).
+-  Our [governance](GOVERNANCE.md).
+-  Our [mission and values](MISSION_AND_VALUES.md).
+-  Our [benchmarking guide](BENCHMARKS.md).
+-  [PEP8](https://www.python.org/dev/peps/pep-0008/) for Python style.
+-  [PEP257](https://www.python.org/dev/peps/pep-0257/) and the
+   [NumPy documentation guide](https://docs.scipy.org/doc/numpy/docs/howto_document.html)
+   for docstring conventions.
+-  [`pre-commit`](https://pre-commit.com) hooks for autoformatting.
+-  [`black`](https://github.com/psf/black) autoformatting.
+-  [`flake8`](https://github.com/PyCQA/flake8) linting.
+-  [#napari on image.sc](https://forum.image.sc/tags/napari).
+-  [#napari](https://twitter.com/search?q=%23napari&f=live) and [@napari_imaging](https://twitter.com/napari_imaging) on twitter.
+-  [napari zulip](https://napari.zulipchat.com/) community chat channel.
+
+You are not required to monitor the social resources.
+
+Where possible we prefer to point people towards asynchronous forms of communication
+like forum posts and github issues instead of realtime chat options as they are easier
+for a global community to consume.
+
+We also have a private mailing list for core developers
+`napari-core-devs@googlegroups.com` which is sparingly used for discussions
+that are required to be private, such as voting on new core members.
+
+## Inviting New Core Members
+
+Any core member may nominate other contributors to join the core team.
+While there is no hard-and-fast rule about who can be nominated; at a minimum,
```

Comment:
fixed - also put in `ideally` instead of `at a minimum` as I don't think we need to be so strict, especially after saying `there is no hard-and-fast rule`